library/build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
 
     api platform("com.squareup.okhttp3:okhttp-bom:$okhttpVersion")
     api "com.squareup.okhttp3:okhttp"
+    api "com.squareup.okhttp3:okhttp-brotli"
```

Comment:
I'm not sure if we need Brotli package from OkHttp as it only adds an interceptor for Brotli compression. I think [`org.brotli:dec`](https://mvnrepository.com/artifact/org.brotli/dec/0.1.2) should be enough.

Also, it doesn't need to be `api`. We don't expose any types.

Reply:
Yeah, correct. Overlooked that `BrotliInputStream` is part of another library.